weblate.md

```diff
@@ -1,83 +1,85 @@
+Before adding a translation check that the string you are translating matches a word of the language's glossary. If you are translating a word and that word is in the glossary, you should always use the suggested translation. (More info in the *What's a glossary? How to use it?* section). If there is no glossary available, the section in the screenshot will not appear.
 
 ### 3. Use the Translation Memory
 ![translation_memory](/media/weblate/translation-memory.png)
 
-Check if there is a similar word/phrase in the translation memory (more accurate strings will have their bar colored green). If you think the suggested string is accurate, copy the string using the given button and submit your suggestion. Keep in mind that accurate suggestons by the translation memory must always be used, if present.  
+Check if there is a similar word/phrase in the translation memory (more accurate strings will have their bar colored green). If you think the suggested string is accurate, copy the string using the given button and submit your suggestion. Keep in mind accurate suggestons by the translation memory must always be used, if present.  
 Check the section *What's a translation memory? How to use it?* for more info.
 
 
 ### 4. Submit your translation and wait for a review
 Click *Suggest string* to submit your translation. Now only needs to be approved, this can happen in two different ways:
 
-- **By vote** Every user have the possibility to vote a suggested translation. When a translation has 1 addition approval (to be added to the automatic approval of the submitter) gets confirmed. To see how voting works check [the section "Review and vote strings"](#review-and-vote-strings)
+- **By vote** Every user has the possibility to vote for or against a suggested translation. When a translation has 1 addition approval (to be added to the automatic approval of the submitter) it gets confirmed. To see how voting works check [the section "Review and vote strings"](#review-and-vote-strings)
 - **By reviewers**: Strings can be confirmed/rejected/edited by a specific figure on Weblate, The "Reviewer". More info about the Reviewer role in the section *[The role of the Reviewer](#the-role-of-the-reviewer)*.
 
 ## General guidelines
 
-- leave placeholders (like `%1`) and HTML tags as they are
-- If the "needs edit" box is ticked, check the translation. If it's ok, unthick the box and "add suggestion"
-- Use gender neutral words when possible
+- Leave placeholders (like `%1`) and HTML tags as they are
+- If the "needs edit" box is ticked, check the translation. If it's OK, unthick the box and "add suggestion"
+- Avoid invoking genders that aren't there
```

Comment:
This is confusing. In any case, i was thinking of deleting this section entirely. People should follow the general guidelines, we don't need to specify them in this guide.